src/lib/cache/subCache.cpp

```diff
@@ -682,7 +682,6 @@ void subCacheItemInsert
   const std::string&            georel
 )
 {
-  LM_W(("KZ: inserting sub in cache. extended == %s", httpInfo.extended? "true" : "false"));
```

Comment:
NTC

Let's do a careful look to next feature/2015_... -> develop PR in order to check that no new LM_W remains.